src/router/index.js

```diff
@@ -15,20 +15,22 @@ const routes = [
     // https://router.vuejs.org/guide/essentials/passing-props.html#function-mode
     path: '/search',
     name: 'Search',
-
     // route level code-splitting
     // this generates a separate chunk (result.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "search-list" */ '../views/Search.vue'),
+    component: () => import(/* webpackChunkName: "search-list" */ '../views/Search'),
   },
   {
-    path: '/search/detail',
+    path: '/search/detail/:fileType',
```

Comment:
We can merge this now but we should add an issue splitting out the detail views for various types to their respective URL's. Some of these viewers require a sizeable chunk of JS (e.g. Video.js) and having lazy-loading on them is more than a luxury.